nolearn/lasagne/visualize.py

```diff
@@ -270,14 +271,23 @@ def get_hex_color(layer_type):
         - layer_type : string
             Class name of the layer
     :returns:
-        - layer_color : string containing a hex color for filling block.
-        - font_color : string containing a hex color for font color.
+        - color : string containing a hex color for filling block.
     """
-    if 'Conv' in layer_type:
-        return '#ff0000', 'cyan'
+    COLORS = ['#4A88B3', '#98C1DE', '#6CA2C8', '#3173A2', '#17649B', \
```

Comment:
It still gives the same error
>C:277, 0: Wrong continued indentation.
                       '#FFBB60', '#FFDAA9', '#FFC981', '#FCAC41', '#F29416',
              |        ^ (bad-continuation)